R/weightedMeanByID.R

```diff
@@ -0,0 +1,86 @@
+#' @title Aggregate different rows assigned to the same ID by calculating a 
+#' weighted mean 
+#'
+#' @description After calculating row means to summarize across replicates, 
+#' values of different rows that are assigned to the same ID are summarized by 
+#' calculating a weighted mean. The weights are the values themselves.
+#' 
+#' @author Fiona Ross
+#' 
+#' @param SE \code{SummarizedExperiment} object that contains an assay with 
```

Comment:
A couple of suggestions:
- Remove the empty lines between the different '@param' lines (to make it more apparent that they belong together)
- Formulate the description of an argument more like an full sentence (rather than {type} + {description}). E.g. 'A \code{SummarizedExperiment} object that...', or 'A \code{character} scalar representing the name of the assay...'